Feature/schedule view

goccert25 opened the pull request:
Resolves #13 
Pull request implements accordion menu based off of Andy's wireframe. In addition, adds in firebase implementation so that the schedule is dynamically generated off of firebase database. Additional styling to the tab bar at the bottom also has been included. 

Still need to implement persistent memory in app so that in case of no internet access, users still have the latest information.

Commit:
Add a rudimentary event element for schedule view.

Changes are in ScheduleScene.js. Dropdown expansion has
yet to be implemented. function _onPressExpand() has no utility.

Commit:
Add trivial dropdown functionality to schedule menu.

Menu is trivial and barebones. Still needs ListView
implemented so as to allow dynamic schedule generation
based on database.

AyeAndy committed:
new schedule view using accordion library

AyeAndy committed:
Merge branch 'develop' into feature/schedule-view

Commit:
modify to automatically install accordion library

AyeAndy committed:
Changed some fonts and spacings of the schedule

Commit:
Merge branch 'feature/schedule-view' of https://github.com/TerrapinFreelancers/bitcamp-app into feature/schedule-view

AyeAndy committed:
aesthetic check up

Commit:
commit with run line errors

AyeAndy committed:
aesthetic

Commit:
currently working version with chevron dropdown working correctly

Commit:
Merge branch 'feature/schedule-view' of https://github.com/TerrapinFreelancers/bitcamp-app into feature/schedule-view

Commit:
fix chevron drop down arrow

Commit:
Merge branch 'develop' into feature/schedule-view

jsjqian committed:
Alter .gitignore to ignore vscode files

jsjqian commented:
LGTM except for the white space above the navbar on Android. Does anyone know why this is occurring? @AyeAndy

Commit:
fix large margin at top of android, highlighting on date press in
schedule view, and changes schema of firebase to be more intuitive

jsjqian commented:
Everything looks good on my phone

Commit:
add persistent data, so without internet connection
schedule view will display latest pull

jdai8 committed:
Refactor schedule scene

Fixes android accordion animation
Removes the empty section headers warning
Combines AccordionMenu with ScheduleScene
Simplifies Accordion component by hardcoding props
